scripts/remove-github-action-runner.sh

```diff
@@ -2,15 +2,29 @@
 
 # This script can be used to remove a Github Action Runner on an openSUSE or SLE
 # distro. It will unconfigure the service and unregister the runner.
+# Copy the script to runner:/home/<user> and run it as <user>.
+# It requires GITHUB_REPOSITORY_URL (https) and GITHUB_RUNNER_TOKEN to be set
+# e.g. export GITHUB_REPOSITORY_URL=https://github.com/epinio/epinio
+# and  export GITHUB_RUNNER_TOKEN=<current token from github settings/actions/runners/new>
+# Note: You can use the same token to add or remove multiple runners,
+#       while it will expire after 1h.
 
 set -e
 
+if [ -z "$GITHUB_REPOSITORY_URL" ]; then
+  echo "The variable GITHUB_REPOSITORY_URL is empty. Exiting"
+  exit 1
+fi
+
 if [ -z "$GITHUB_RUNNER_TOKEN" ]; then
   echo "The variable GITHUB_RUNNER_TOKEN is empty. Exiting"
   exit 1
 fi
 
+REPOSITORY_NAME=$(echo $GITHUB_REPOSITORY_URL | cut -d '/' -f 4- | sed -e 's|/$||g' -e 's|/|-|g')
+ACTIONS_RUNNER_SERVICE=actions.runner.$REPOSITORY_NAME.`hostname`.service
+
 cd actions-runner
-sudo systemctl stop actions.runner.epinio-epinio.`hostname`.service
+sudo systemctl stop $ACTIONS_RUNNER_SERVICE
 sudo ./svc.sh uninstall
 ./config.sh remove --token $GITHUB_RUNNER_TOKEN
```

Comment:
Not your change, but I wonder if GITHUB_RUNNER_TOKEN can ever contain a string like `/*/*/*/*/../../../../*/*/*/*/../../../../*/*/*/*`. Quoting it might be safer ;)

Reply:
Good jint. Will incorporate quoting where needed in the scripts.